.github/workflows/build.yml

```
@@ -23,6 +23,18 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v4
 
+      # This is required to confirm that GHA will use the right version of Yarn
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Check yarn version
+        run: |
+          if [ `yarn --version` != `cat package.json | jq .packageManager -r | cut -c 6-` ]; then
```

```suggestion
          if [ "`yarn --version`" != "`cat package.json | jq .packageManager -r | cut -c 6-`" ]; then
```

I would feel more comfortable having quotes around these values, just in case something breaks and one of them becomes empty or otherwise changes in a way that could cause a shell error.